src/renderer/components/message/MessageList.js

```diff
@@ -65,6 +65,9 @@ export default function MessageList({
 
   useEffect(() => {
     isFetching.current = false
+    
+    const composerTextarea = refComposer.current.childNodes[1]
```

Comment:
this looks like a hack to me... how about making a focus function inside of the composer component and calling it here?

Reply:
You can only bind functions to class components.

Reply:
I vote for making the compose a class, it makes sense, also when we have more advanced compositing going on it might be useful to have more control over that part.

Reply:
like `:emoji:`- and user-mention-(dropdown|prediction) 
or fancy richtext/markdown previews like the ones discord introduced (though that will be far in the future I believe)

Reply:
I'm fine with making it a class, whatever floats the boat, but this pr is to fix a bug and not to rewrite something.

Reply:
okey.. but let's don't forget it :wink: